astropy/io/ascii/daophot.py

```diff
@@ -142,12 +146,22 @@ def extract_keyword_line(self, line):
             return m.group('name'), keyword_dict
 
     def get_cols(self, lines):
-        """Initialize the header Column objects from the table ``lines`` for a DAOphot
+        """
+        Initialize the header Column objects from the table ``lines`` for a DAOphot
         header.  The DAOphot header is specialized so that we just copy the entire BaseHeader
         get_cols routine and modify as needed.
-
-        :param lines: list of table lines
-        :returns: list of table Columns
+        
+        
+        
+        Parameters
+        ----------
+        lines: list 
+            List of table lines
+            
+        Returns
+        ----------
+        col : list
+            List of table Columns
```

Comment:
If these are ``astropy.table.Column`` objects then they should be linked there (e.g. `~astropy.table.Column`)